apps/frontend/src/app/3_organisms/FastBtcDialog/components/SendFlow/components/AmountForm.tsx

```diff
@@ -85,6 +80,22 @@ export const AmountForm: React.FC = () => {
     [set, value],
   );
 
+  const maxAmount = useMemo(() => {
+    return Math.min(Number(formatUnits(limits.max, 8)), Number(maxRbtcBalance));
+  }, [limits.max, maxRbtcBalance]);
```

Comment:
Please use BigNumber instead of Number when working with balances than are being sent to the transactions, otherwise some decimal places will be lost.